index.es6

```diff
@@ -19,6 +25,11 @@ export default function createMiddleware(dsn, cfg={}) {
   }
 
   return store => next => action => {
+    const {
+      actionTransformer = identity,
+      stateTransformer = identity,
+      logger = console.error
```

Comment:
... or even better:

    logger = (err) => console.log('[redux-raven-middleware] Reporting error to Sentry:', err)

So we can handle the real exception.